SubjuGator/command/subjugator_launch/readme.md

```diff
@@ -9,3 +9,11 @@ sub8 launch files and scripts
 Each driver can be *turned off* If you are running this launch file alone, each of these defaults to false. This is because the only conditions that I expect someone to run the nav\_box file outside of a launch file is for debugging, and it is thus more convenient to default to off.
 
 This code is almost entirely inherited from previous subs, and is authored almost entirely by Forrest Voight.
+
+# PID Debugging Data Collection
+
+**NOTE:** This launch file needs the sub launch file (`sub8.launch`) running in order to function properly.
+
+    roslaunch subjugator_launch bag_debugging_controller.launch prefix_name:="my_prefix"
+
+This launch file records a bag file containing adaptive controller `pose_error`, `twist_error`, `adaptation`, `dist`, and `drag` data along with `wrench`, `trajectory`, `odom`, `imu/data_raw`, and `dvl/range` data. This data is useful in determining if the PID values are getting better or worse using simulations. The bag file was written by Andres Pulido.
```

Comment:
Unfortunately, the READMEs of various packages are almost never read. Instead, add your documentation into a new or existing doc under the `docs/` folder, where it will be converted to HTML and added to our documentation site.

Reply:
876436d has a new documentation page. With this information and a bit more I added in from testing sessions.